opentelemetry-sdk/tests/resources/test_resources.py

```diff
@@ -492,6 +492,16 @@ def test_invalid_key_value_pairs(self):
             resources.Resource({"k": "v", "k2": "v2", "foo": "bar=baz"}),
         )
 
+    def test_multiple_with_url_decode(self):
+        detector = resources.OTELResourceDetector()
+        os.environ[
+            resources.OTEL_RESOURCE_ATTRIBUTES
+        ] = "key=value%20test%0A, key2=value+%202"
```

Comment:
consider using a url decoder instead of the result

Reply:
Fixed, Thank you for the comment

Reply:
@nemoshlag 
Is your comment resolved?

Reply:
indeed, thanks